backend/db/initialize.py

```diff
@@ -52,13 +54,31 @@ def initialize(
     - https://docs.sqlalchemy.org/en/20/dialects/mysql.html
     """
     with get_db_connection(local=local) as con:
-        if _create_db:
-            create_db(con)  # causing error. don't need it at the moment anyway
+        if create_db:
+            create_database(con, schema)  # causing error. don't need it at the moment anyway
         if download:
             download_artefacts(force_download_if_exists=download_force_if_exists)
         seed(con, schema, clobber)
         indexes_and_derived_tables(con, schema) #, start_step=30)
 
 
+def cli():
+    """Command line interface"""
+    parser = ArgumentParser(description='Initializes DB.')
```

Comment:
Adde CLI to `initialize()`. This helped me initialize it locally with the right settings without altering the existing code.